modules/eventadd.php

```diff
@@ -401,6 +401,8 @@
     }
     if (!isset($eventticketid)) {
         $event['helpdesk'] = ConfigHelper::checkConfig('phpui.default_event_ticket_assignment') ? 'new' : 'none';
+    } elseif ($eventticketid == '-1') {
+        $event['helpdesk'] = 'none';
     }
```

Comment:
A to nie jest przy okazji jakiś "fixup", który powinien trafić również do którychś gałęzi stable?

Reply:
Ja do stable wrzucam tylko i wyłącznie konieczne rzeczy.

Reply:
Masz wydzielony commit z tą zmianą możesz zrobić cherry-pick do stable.